mlir/test/Conversion/UBToEmitC/convert-ub-to-emitc-unsupported.mlir

```diff
@@ -0,0 +1,15 @@
+// RUN: mlir-opt -convert-ub-to-emitc -split-input-file -verify-diagnostics %s
+
+func.func @poison_memref() {
+  // expected-error @+1 {{failed to legalize operation 'ub.poison'}}
+  %0 = ub.poison : memref<i32>
+  return
+}
+
```

Comment:
Can you add a test containing a `vector`? I assume it will (correctly) not be lowered, but it seems to be the only special case in the `ub.poison` spec.

Reply:
Yes; I don't fully understand how the poison attribute works, but it looks like my example is accepted.